diff --exclude='*~' -ruN mkcal-0.3.11.orig/src/extendedstorage.cpp mkcal-0.3.11/src/extendedstorage.cpp
--- mkcal-0.3.11.orig/src/extendedstorage.cpp	2011-09-21 11:26:08.107577157 +0300
+++ mkcal-0.3.11/src/extendedstorage.cpp	2011-09-21 11:30:50.318911824 +0300
@@ -753,7 +753,7 @@
     }
   }
 #else
-  Q_UNUSED( incidence );
+  Q_UNUSED( incidences );
 #endif
 }
 
diff --exclude='*~' -ruN mkcal-0.3.11.orig/src/src.pro mkcal-0.3.11/src/src.pro
--- mkcal-0.3.11.orig/src/src.pro	2011-09-21 11:26:08.108577164 +0300
+++ mkcal-0.3.11/src/src.pro	2011-09-21 11:27:21.022215481 +0300
@@ -43,7 +43,7 @@
     /usr/include/dbus-1.0 \
     /usr/include/qt4/QtDBus
 
-DEFINES += MEEGO UUID TIMED_SUPPORT #MKCAL_TRACKER_SYNC
+DEFINES += MEEGO UUID #MKCAL_TRACKER_SYNC
 LIBS += -lQtDBus \
     -lsqlite3 \
     -luuid \
